lib/expr.ex

```diff
@@ -656,6 +658,104 @@ defmodule AshSql.Expr do
     )
   end
 
+  defp default_dynamic_expr(
+         query,
+         %Has{arguments: [left, right], embedded?: pred_embedded?},
+         bindings,
+         embedded?,
+         acc,
+         _type
+       ) do
+    if bindings.sql_behaviour.ilike?() do
```

Comment:
We should not assume postgres because of `ilike?`. Lets add a new behavior function here that tells us we can use "equals any"

Reply:
do you want me to name it `equals_any?` , hardest problem in Software engineering is naming things